### PR TITLE
feat: add support version 1.8

### PR DESCRIPTION
**Pull request by chobbs**
This PR contains fixes for the fault domain support limits as well as updated support for influxdb enterprise version 1.8

### CODE REVIEW

File: src/machines/chronograf-resources.json [hunk 1]
```diff
@@ -33,13 +26,10 @@
...
         },
-        "influxTags": {
+        "networkSettings": {
             "type": "object",
-            "defaultValue": {
-                "provider": "influxdata"
-            },
             "metadata": {
-                "description": "Unique identifiers to allow the Azure Infrastructure to understand the origin of resources deployed to Azure. You do not need to supply a value for this."
+                "description": "Network settings to deploy on"
```
Comment (gunnaraasen) [inline src/machines/chronograf-resources.json:32]: Was there some text cut off here?

File: src/machines/chronograf-resources.json [hunk 2]
```diff
@@ -201,5 +191,10 @@
             ]
         }
     ],
-    "outputs": {}
+    "outputs": {
```
Comment (gunnaraasen) [inline src/machines/chronograf-resources.json:194]: Nice!

File: src/scripts/enterprise-configure.sh [hunk 3]
```diff
@@ -248,55 +207,88 @@ EOF
...
   fi
 }
+telegraf()
+{
+  #generate and stage new telegraf configuration file
+  log "[configure_telegraf] generating new telegraf configuration file at ${TELEGRAF_CONFIG_FILE}"
+
+    touch "${TELEGRAF_CONFIG_FILE}"
+    if [ $? -eq 0 ]; then
+      cat > "${TELEGRAF_CONFIG_FILE}" <<-EOF
```
Comment (gunnaraasen) [inline src/scripts/enterprise-configure.sh:230]: I think we actually include a Telegraf config file in the images. Just need to copy the config over ([file](https://github.com/influxdata/azure-resource-manager-influxdb-enterprise/blob/master/packer/config/telegraf-data.conf) and [path](https://github.com/influxdata/azure-resource-manager-influxdb-enterprise/blob/master/packer/scripts/setup-telegraf.sh#L10)).

File: src/scripts/influxdb-install.sh [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+#########################
+
+#Script Parameters
+OSS_VERSION="1.8.0"
+
+#########################
+# Installation 
+#########################
+
+install_influxdb()
```
Comment (gunnaraasen) [inline src/scripts/influxdb-install.sh:81]: When we decide to update the monitor to InfluxDB v2, I think we should publish a new image for the monitor node with InfluxDB pre-installed. This is good for now.

File: src/machines/chronograf-resources.json (hunk 1 above)
Comment (chobbs) [reply src/machines/chronograf-resources.json:32]: hey @gunnaraasen -- good catch, yup i accidentally cut of the descriptions -- will fill out

**Pull request merged**